framework/src/materials/PiecewiseLinearInterpolationMaterial.C

```diff
@@ -0,0 +1,87 @@
+/****************************************************************/
```

Comment:
Any thoughts on the name for this?  @permcody? There are a couple of other materials of this nature in this directory with names that start with 'Generic', but I didn't include that to keep the name shorter.